src/apiCalls.js

```diff
@@ -1,71 +1,70 @@
 export const getAllMovies = async () => {
-  const response = await fetch('http://localhost:3000/api/v1/movies')
+  const response = await fetch('https://indecision-maker.herokuapp.com/api/v1/movies')
```

Comment:
I noticed that there is a new change from `HTTP` to `HTTPS`, is this related to the server being hosted in Heroku?

Reply:
Great question, Orlando! I'd love to know, too.